chart-service/chart-generator.js

```
@@ -170,6 +325,66 @@
+      if (!chart.options.plugins.title.font) {
+        chart.options.plugins.title.font = { family: "'DejaVu Sans', 'Liberation Sans', 'Fallback Font', sans-serif" };
+      }
+      
+      // Set font family for legend
+      if (!chart.options.plugins.legend.labels && chart.options.plugins.legend.display !== false) {
+        chart.options.plugins.legend.labels = { font: { family: "'DejaVu Sans', 'Liberation Sans', 'Fallback Font', sans-serif" } };
+      }
+    }
+    
+    // Set global defaults for scales
+    if (chart.options && chart.options.scales) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)